stoobly_agent/app/cli/helpers/endpoints_import_service.py

```diff
@@ -32,7 +32,9 @@ def import_endpoints(context: EndpointsImportContext):
 
         endpoint_id = res.json()['id']
         try:
-            res = import_component_names(context.project_id, endpoint_id, endpoint, context.resources)
+            body_param_parents: Dict[int, int] = {}
```

Comment:
Do we need to drill this a few methods i.e. `import_component_names` and `process_import`?

Reply:
No we don't, I've moved parent instantiation into `import_component_names`